weed/storage/disk_location.go

```
@@ -385,24 +390,21 @@ func (l *DiskLocation) reconcileCompactStates() {
 	}
 	pending := make(map[volKey]bool)
 	collect := func(dir string) {
-		entries, err := os.ReadDir(dir)
-		if err != nil {
-			return
-		}
-		for _, entry := range entries {
+		eachDirEntry(dir, func(entry os.DirEntry) bool {
```

golangci-lint (2.12.2)

[error] Error return value is not checked

(errcheck)